TreesForTravel/screens/DetailsScreen.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { ScrollView, StyleSheet, View, Text, Button, Image } from 'react-native';
+import { ExpoLinksView } from '@expo/samples';
+
+import { connect } from 'react-redux';
+import { getFlights, deleteFlight } from '../Redux/actions.js';
+
+import DataFunctions from '../dataProcessing/functions.js'
+
+class DetailsScreen extends React.Component {
+
+  constructor(props) {
+     super(props);
+   }
+
+   deleteFlight = async (id) => {
+     fetch('http://192.168.1.187:3000/flights/' + id, {method: 'DELETE'})
```

Comment:
It's better to centralize this value, then if you change it (preparing for production) you only have to change it once.